fortify-issue-manager/src/main/java/com/microfocus/security/automation/fortify/issue/tracker/OctaneBugTrackerSettings.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.microfocus.security.automation.fortify.issue.tracker;
+
+import java.util.Map;
+
+final class OctaneBugTrackerSettings extends BugTrackerSettings
+{
+    private final String sharedSpaceId;
+    private final String workspaceId;
+
+    public OctaneBugTrackerSettings(
+            final String username,
+            final String password,
+            final String apiUrl,
+            final String sharedSpaceId,
+            final String workspaceId,
+            final Map<String, String> proxySettings)
+    {
+        super(username, password, apiUrl, proxySettings);
+        this.sharedSpaceId = sharedSpaceId;
+        this.workspaceId = workspaceId;
```

Comment:
Are these always numbers in Octane?  I'm wondering if they should be parsed to check that numbers have been passed (which would avoid them being used to inject anything into the urls later).